.github/workflows/release.yml

```diff
@@ -0,0 +1,37 @@
+name: release
+
+on:
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Java
+      uses: actions/setup-java@v3
+      with:
+        distribution: 'temurin'
+        java-version: '11'
+
+    - name: Build with Maven
+      run: ./mvnw package -DskipTests
+
+    - name: Set up Apache Maven Central
+      uses: actions/setup-java@v3
+      with: # running setup-java again overwrites the settings.xml
+        distribution: 'temurin'
+        java-version: '11'
```

Comment:
Same comment here about caching.

Do you have a resource that explains why we want to overwrite the `settings.xml`?

Reply:
Found an example here at work that confirmed this, so I simplified this.